mlptrain/configurations/configuration.py

```diff
@@ -39,6 +39,8 @@ def __init__(self,
         self.energy = Energy()
         self.forces = Forces()
 
+        self.plumed_coordinates = None
```

Comment:
this needs a type hint for what it is, and probably (as much as I don't like them) an inline comment
```suggestion
        self.plumed_coordinates: Optional[np.ndarray] = None  # whatever the array is!
```